src/Tizen.Wearable.CircularUI.Forms.Renderer/CircularUI.cs

```diff
@@ -44,5 +61,23 @@ public static void Init(string apiKey)
 
             Init();
         }
+
+        public static void Init(CoreApplication context)
+        {
+            var resPath = context?.DirectoryInfo?.Resource;
+            if (string.IsNullOrEmpty(resPath))
+                ThemeLoader.Initialize(resPath);
+
+            Init();
+        }
+
+        public static void Init(InitOptions options)
+        {
+            var resPath = options.Context?.DirectoryInfo?.Resource;
+            if (string.IsNullOrEmpty(resPath))
```

Comment:
is It correct code ?  resPath should be neither null nor empty

Reply:
haha :eyes: